free-2.md

```diff
@@ -58,12 +94,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Almost 99% of the work would be on the system looking up recipes to recommend. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project because of the web scraping would be ambitious for the semester. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a bad idea since it doesn't allow for that much user expressiveness and
```

Comment:
Agreed. There doesn't seem to be much in terms of syntactical variety for the user. Basically just providing inputs for a program.